python/ray/tests/test_dashboard.py

```diff
@@ -39,6 +93,64 @@ def test_dashboard(shutdown_only):
                     f"Dashboard output log: {out_log}\n")
 
 
+@pytest.mark.skipif(
+    sys.platform.startswith("win"), reason="No need to test on Windows.")
+def test_k8s_cpu():
```

Comment:
Just realized this might not be the best place for this new test.
Where in the CI does test_dashboard.py run? (It's not specified in tests/BUILD)